shared/src/main/scala/govariants/Board.scala

```diff
@@ -20,25 +21,27 @@ class Board(val size: Int) {
     }
   }
 
-  def display() = {
-    print("  ")
+  def as_string(): String = {
```

Comment:
It's great to switch to a pure function. We can probably even switch to `@override def toString`.